orga/app/templates/components/pagination-control.hbs

```diff
@@ -0,0 +1,38 @@
+<div class="pagination-control">
+  <div class="page-size">
+    <div class="page-size__label">Voir</div>
+    <div class="page-size__choice">
+      <select class="select" onchange={{action 'changePageSize' value='target.value'}}>
+        <option value="10" selected="{{if (eq pagination.pageSize 10) true}}">10</option>
+        <option value="25" selected="{{if (eq pagination.pageSize 25) true}}">25</option>
+        <option value="50" selected="{{if (eq pagination.pageSize 50) true}}">50</option>
+        <option value="100" selected="{{if (eq pagination.pageSize 100) true}}">100</option>
+      </select>
+    </div>
+  </div>
+  <div class="page-navigation">
+    <div class="page-navigation__arrow page-navigation__arrow--previous {{if (eq pagination.page 1) "page-navigation__arrow--disabled"}}">
+      {{#link-to "authenticated.campaigns.details.participants"
```

Comment:
Dans un composant au nom qui semble réutilisable (pagination-control), je pense que le link-to qui est lié à son utilisateur (campaigns.details.participants) fait un peu bizarre. Est-ce qu'on pourrait lui passer en paramètre ?

Reply:
oui

Reply:
tout à fait, c'est fait